client/src/components/settings/Certification.test.js

```diff
@@ -15,75 +15,62 @@ describe('<certification />', () => {
   // shallow rendering does not render children component
   // form buttons are not included in shallow render
   it('Should render show cert button for claimed legacy cert', () => {
-    const { container } = renderWithRedux(
-      <CertificationSettings {...defaultTestProps} />
-    );
+    renderWithRedux(<CertificationSettings {...defaultTestProps} />);
 
     expect(
-      container.querySelector(
-        'a[href="/certification/developmentuser/legacy-data-visualization"]'
-      )
-    ).toHaveTextContent('buttons.show-cert');
-  });
-
-  it('Should link show cert button to the claimed legacy cert', () => {
-    const { container } = renderWithRedux(
-      <CertificationSettings {...defaultTestProps} />
+      screen.getByRole('link', {
+        name: 'buttons.show-cert'
+      })
+    ).toHaveAttribute(
+      'href',
+      '/certification/developmentuser/legacy-data-visualization'
     );
-
-    expect(
-      container.querySelector(
-        'a[href="/certification/developmentuser/legacy-data-visualization"]'
-      )
-    ).toBeInTheDocument();
   });
```

Comment:
@huyenltnguyen  I found this test and the one above it similar so I had removed the test.. Correct me if I am wrong..